### PR TITLE
Enabling csharp webhook and proxy heartbeat functions

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
The csharp functions I worked on some time ago had not been enabled. Doing in this PR. 

## PR Checklist
* [ ] Applies to work item: #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

**Review by chkeita — APPROVED** [review]

File: src/deployment/deploy.py [hunk 1]
```diff
@@ -1147,6 +1147,10 @@ def expand_agent(f: str) -> List[str]:
                             ],
                             env=dict(os.environ, CLI_DEBUG="1"),
                         )
+
+                        if function_name == "queue_proxy_update":
```
Comment (chkeita) [inline src/deployment/deploy.py:1151]: why the change of name ?

Comment (nharper285) [reply src/deployment/deploy.py:1151]: I changed it awhile back because I thought it was a more accurate function name. The function _only_ handles the heartbeat updates.

Comment (chkeita) [reply src/deployment/deploy.py:1151]: i propose we keep it as is and change it after the migration is done.

Comment (chkeita) [reply src/deployment/deploy.py:1151]: this will minimize the custom code we need to support the rename

Comment (nharper285) [reply src/deployment/deploy.py]: Done.
